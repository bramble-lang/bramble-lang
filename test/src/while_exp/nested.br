fn my_main() -> i64 {
    let mut x: i64 := 0;
    while (x <= 5) {
        let mut y: i64 := 0;
        while (y <= 2) {
            project::std::io::write("y: ");
            project::std::io::writei64ln(y);
            mut y := y + 1;
        };
        project::std::io::write("x: ");
        project::std::io::writei64ln(x);
        mut x := x + 1;
    };
    return 0;
}