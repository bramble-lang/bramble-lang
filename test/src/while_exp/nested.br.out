y: 0
y: 1
y: 2
x: 0
y: 0
y: 1
y: 2
x: 1
y: 0
y: 1
y: 2
x: 2
y: 0
y: 1
y: 2
x: 3
y: 0
y: 1
y: 2
x: 4
y: 0
y: 1
y: 2
x: 5
